thingworx-import/Entities.xml

```diff
@@ -678,11 +678,162 @@ TyqZNc0vshhVwE/OK4OsFVTXATO/uu5FZ5fgfXfotCBIJBTXfJuCCwGuOahZo/N+tl7BN2PormEP
+let m2WebCompleteURL = encodeURI(m2WebHost + m2WebCommand + m2WebCredentials + m2WebEwonCredentials);
+let params = {
+    url: m2WebCompleteURL /* STRING */ ,
+};
+
+// Perform request to M2Web and recieve response body in the following format:
+/*
+   "TagId";"TagName";"Value";"AlStatus";"AlType";"Quality"
+   2;"DEMOSTOP";0;0;0;65472
+   3;"DEMOSCENARIO";2;0;0;65472
+   4;"DEMOINUSE";1;0;0;65472
+   5;"LiftStation-A01-FLOW";8770;0;0;65472
+   6;"LiftStation-A01-PWR";1;0;0;65472
+   7;"LiftStation-B01-FLOW";9160;0;0;65472
+   8;"LiftStation-B01-PWR";1;0;0;65472
+*/
+var rawEbdResult = me.tagReadCache;
+if (useCached && me.tagReadCache.length <= 0) {
+    logger.warn("A request to read cached tag data was made, but no cached data was found! Reading via M2Web.");
+    rawEbdResult = Resources['ContentLoaderFunctions'].GetText(params);
+    me.tagReadCache = rawEbdResult;
+} else if (!useCached) {
+    rawEbdResult = Resources['ContentLoaderFunctions'].GetText(params);
+    me.tagReadCache = rawEbdResult;
+}
+
+// Split response body in to individual lines (filtering out empty entries)
+var rawEbdLines = rawEbdResult.split(/\r\n|\n|\r/).filter(function (x) { return x != ""; });
+
+// Loop through each response body line and add to info table
+for (var i = 1, rawEbdLineSegments; i < rawEbdLines.length; i++) {
+    // Split line to individual fields
+    rawEbdLineSegments = rawEbdLines[i].split(';');
+
+    // Create new info table entry with field values
+    let newInfoTableRow = {
+        TagId: parseInt(rawEbdLineSegments[0]), // INTEGER
+        TagName: rawEbdLineSegments[1], // STRING
+        Value: rawEbdLineSegments[2], // STRING
+        AlStatus: parseInt(rawEbdLineSegments[3]), // INTEGER
+        AlType: parseInt(rawEbdLineSegments[4]), // INTEGER
+        Quality: parseInt(rawEbdLineSegments[5]) // INTEGER
+    };
+
+    // Add new entry to info table
+    m2WebResultInfoTable.AddRow(newInfoTableRow);
+}
+
+// Return resulting info table
+let result = m2WebResultInfoTable;]]></code></Row></Rows></ConfigurationTable></ConfigurationTables></ServiceImplementation><ServiceImplementation description="" handlerName="Script" name="ReadBooleanTagLiveGenericService"><ConfigurationTables><ConfigurationTable dataShapeName="" description="" isMultiRow="false" name="Script" ordinal="0"><DataShape><FieldDefinitions><FieldDefinition baseType="STRING" description="code" name="code" ordinal="0"/></FieldDefinitions></DataShape><Rows><Row><code><![CDATA[// Read live tag values for Ewon
+let m2WebResultInfoTable = me.ReadAllTagsLiveGenericService({
+    ewonName: ewonName /* STRING */,
+    useCached: useCached /* BOOLEAN */
+});
+
+let m2WebResultInfoTableRowMatch = m2WebResultInfoTable.Find({
+    TagName: "\""+tagName+"\""
+});
+
+let result = null;
+try {
+    // Get value and cast to boolean
+    let rawValueBooleanInt = parseInt(m2WebResultInfoTableRowMatch.Value);
+    result = (rawValueBooleanInt == 1);
```

Comment:
Is there any error logging available? If for some reason rawValueBooleanInt was 2 it might be nice to have that logged. not that that would happen under normal circumstances.

Reply:
added